pkg/kubernetes/pod/v1alpha1/pod.go

```diff
@@ -142,3 +142,35 @@ func IsNil() Predicate {
 func (p *Pod) GetAPIObject() *corev1.Pod {
 	return p.object
 }
+
+// FromList created a PodList with provided api podlist
+func FromList(pods *corev1.PodList) *PodList {
+	pl := ListBuilderForAPIList(pods).List()
+	return pl
+}
+
+// GetNodeNames returns the nodes on which pods are running
+func (pl *PodList) GetNodeNames() []string {
+	nodeNames := []string{}
+	for _, p := range pl.items {
+		nodeNames = append(
```

Comment:
Does this throw scopelint error?
Do we have simple & specific UT?

Reply:
No I will add UT for this